docs/assets/tabs.scss

```diff
@@ -0,0 +1,101 @@
+li a[href="/"],
+li a[href="/es"],
+li a[href="/ja"],
+li a[href="/pt"],
+li a[href="/zh-Hant"],
+li a[href="/zh-Hans"] {
+  @apply -ml-2 mb-2 text-gray-500 uppercase tracking-wider font-bold text-sm;
+
+  @screen lg {
+     @apply text-xs;
+  }
+}
+
+$background: #43536a;
```

Comment:
*suggestion*: I would access these through tailwind theme rather than hard coding

Reply:
Yes, totally agree to move it into `tailwind.config.js`